code/__DEFINES/shuttles.dm

```diff
@@ -21,6 +21,7 @@
 #define SHUTTLE_HEIGHT_TOO_LARGE "height_too_large"
 #define SHUTTLE_ALREADY_DOCKED "we_are_already_docked"
 #define SHUTTLE_SOMEONE_ELSE_DOCKED "someone_else_docked"
+#define SHUTTLE_DOCK_OBSTRUCTED "shuttle_dock_obstructed"
```

Comment:
What check is that? 

Using default shuttle code I could drive the shuttle right through the station, if there was another check it sure wasn't working. Also the assault pod and asteroid shuttle wouldn't be able to land with this check. That's why I made the actual check based on a new 'safety' var so perhaps other player-controlled shuttles could be added without the risk of people smashing them into the armory every time.